File: Group 57/level 35/Homework/hw3.py
# 3) შექმენით ფუნქცია რომელიც მომხმარებელს შეეკითხება ორ რიცხვს შემდეგ კი მათ დაუმატებთ ერთმანეთს.

def add_two_numbers():
    num1 = int(input("Enter the first number: "))
    num2 = int(input("Enter the second number: "))
    result = num1 + num2
    print("The sum of both numbers is: " + str(result))

add_two_numbers()

File: Group 57/level 143/Homework/hw3.js
// 3)მომხმარებელს შემოატანინე:

// პირველი რიცხვი

// ოპერატორი (+, -, *, /)

// მეორე რიცხვი

// switch–case გამოყენებით შეასრულე ოპერაცია.
// თუ ოპერატორი არასწორია → "არასწორი ოპერატორი".

const num1 = parseFloat(prompt("first number:"));
const operator = prompt("operator (+, -, *, /):");
const num2 = parseFloat(prompt("second number:"));

switch (operator) {
    case "+":
        console.log(num1 + num2);
        break;
    case "-":
        console.log(num1 - num2);
        break;
    case "*":
        console.log(num1 * num2);
        break;
    case "/":
        console.log(num1 / num2);
        break;
    default:
        console.log("Invalid operator");
}